src/openzaak/components/catalogi/utils.py

```diff
@@ -43,6 +43,39 @@ def has_overlapping_objects(
     return False
 
 
+def has_overlapping_objects_with_concepts(
```

Comment:
I don't quite understand why we need two almost the same functions applied for the same model. 
Let's just change logic of existing `has_overlapping_objects` function including `concept` there

This will also take card of overlapping validation in API. Now API and Admin have different logic

Reply:
The original idea was that it only checks if concept exists once because you need to filter on it.

But I have combined both functions